Week-3/Assignment1/index.js

```diff
@@ -0,0 +1,31 @@
+function fib( num ){
+    if( num < 2){
+        return num ;
+    }
+    return fib( num - 1 ) + fib( num - 2 );
+}
+
+function time(val){
+    console.time();
+    console.log(val);
+    console.timeEnd();
+}
+
+function memoize(fun){
+    var cache = new Map();
+    return function (...args){
+        let key = args;
```

Comment:
key conveys it is a single argument.
args conveys there are multiple arguments.